app/views/admin/infobase/_today_menu.html.haml

```diff
@@ -0,0 +1,27 @@
+%div
+  .ui.top.attached.tabular.menu
+    %a.item.active{"data-tab": "first"} First Course
+    %a.item{"data-tab": "main"} Main Course
+    %a.item{"data-tab": "beverages"} Beverages
+
+  .ui.bottom.attached.tab.segment.active{"data-tab": "first", id: 'menu-tab-attached'}
+    = render 'menu_table', menu: infobase.first_course
+
+  .ui.bottom.attached.tab.segment{"data-tab": "main", id: 'menu-tab-attached'}
+    = render 'menu_table', menu: infobase.main_course
+
+  .ui.bottom.attached.tab.segment{"data-tab": "beverages", id: 'menu-tab-attached'}
+    = render 'menu_table', menu: infobase.drinks_course
+
+%br
+
+.ui.two.column.grid{id: 'add-item-form'}
+  .column{id: 'add-item-form-first'}
+    .ui.header
+      Add Item for Today
+    %p Fill out the form below to add new item
+    = render 'menus/form', menu: infobase.new_menu
+  .column
+    %img{alt: "5dcd763ee6174", src: "http://i.piccy.info/i9/c9baf6769ff15bd1f208b6a4c1cbc183/1574706763/27107/1349516/5dcd763ee6174_800.jpg"}
```

Comment:
mmm, what a handful `alt`

```
The required alt attribute specifies an alternate text for an image, if the image cannot be displayed.
The alt attribute provides alternative information for an image if a user for some reason cannot view it (because of slow connection, an error in the src attribute, or if the user uses a screen reader).
```
..and for blind people. You either do not use it, or do it appropriately

Reply:
https://apidock.com/rails/ActionView/Helpers/AssetTagHelper/image_tag